packages/design-system/src/breadcrumbs.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { Breadcrumbs as MuiBreadcrumbs } from "@mui/material";
+import cn from "classnames";
+
+interface BreadcrumbItem {
+  label: string;
+  href: string;
```

Comment:
Tady pridavam poznamku, ze potrebujeme i variantu, ktera nema odkazy, je to jenom vypis. Viz treba (bezodkazova varianta):

https://archiv.volebnikalkulacka.cz/volby/komunalni-2022/582786-brno/navod

Hezky priklad, jak to muze vypadat je i tu (odkazova varianta):
https://app.cesko.digital/projects